src/NServiceBus.Msmq.AcceptanceTests/When_distributing_a_command.cs

```diff
@@ -11,103 +11,178 @@
 
     public class When_distributing_a_command : NServiceBusAcceptanceTest
```

Comment:
What was the reason for changing this test? What does having two different receiver endpoints test that one doesn't?